pyproject.toml

```diff
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "db-client"
-version = "3.9.7"
+version = "3.9.8"
 description = "All things to do with the datamodel and its storage. Including alembic migrations and datamodel code."
 authors = ["CPR-dev-team <tech@climatepolicyradar.org>"]
 license = "Apache-2.0"
 readme = "README.md"
 packages = [{ include = "db_client" }]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.11"
```

Comment:
I'd prefer not to have done this as part of this PR, but we need to to get some of the nice validation, and it needs to be done.

My bad, and I'll deal with any fallout when updating dependencies.